hedera-node/hedera-schedule-service/src/main/java/module-info.java

```diff
@@ -1,10 +1,11 @@
 module com.hedera.node.app.service.schedule {
-    exports com.hedera.node.app.service.schedule;
-
-    uses com.hedera.node.app.service.schedule.ScheduleService;
-
     requires transitive com.hedera.node.app.spi;
     requires transitive com.hedera.node.hapi;
-    requires transitive com.hedera.pbj.runtime;
     requires static com.github.spotbugs.annotations;
+    //
```

Comment:
Yes, spotless removes the line if it's blank.

Reply:
I'm fine having a comment for a line here, but indicating why it's here would definitely be less confusing. I can pretty much guarantee others will remove it as is. At the end of the day, though, it really isn't something to fuss over

Reply:
I wanted a blank line to make the groupings readable.  Unfortunately Spotless has some very unfortunate corners because we didn't really think through some of Palantir's bad habits (and lack of module-info support) before applying it.